packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -155,6 +160,13 @@ export interface IListProps<T = any> extends React.HTMLAttributes<List<T> | HTML
    */
   usePageCache?: boolean;
 
+  /**
+   * Boolean to disable the onAsyncIdle call. Recommended to use when doing jumping through the list.
+   * As the user is moving through the list very quickly, creating lookaheads is heavy and unneccessary.
```

Comment:
_It depends_. The lookahead calculations should stop once the entire set of pages have been traversed. The only time that this _does not_ occur is when the `window` is the scrollable viewport. 

https://github.com/OfficeDev/office-ui-fabric-react/issues/8740

Can you explain your scenario a bit more? We can discuss offline.

Reply:
I removed this change.